src/jit/compiler.h

```diff
@@ -8843,6 +8843,8 @@ void * CompAllocator::ArrayAlloc(size_t elems, size_t elemSize)
 inline
 LclVarDsc::LclVarDsc(Compiler* comp)
     :
+#ifndef _TARGET_X86_
+    // TODO-x86 - Need initialization of the _lvArgReg and _lvOtherArgReg for x86?
```

Comment:
If initializing it to REG_STK is exposing a bug in liveness in x86, why it is not an issue for RyuJIT amd64?  I think we are missing something here. To get to the bottom of it we need to investigate it further rather than working around it temporarily.